lib/modules/manager/nvm/extract.ts

```diff
@@ -4,7 +4,12 @@ import type { PackageDependency, PackageFileContent } from '../types';
 export function extractPackageFile(content: string): PackageFileContent {
   const dep: PackageDependency = {
     depName: 'node',
-    currentValue: content.trim(),
+    currentValue: content
+      .split('\n')
+      // Exclude code comments
+      .filter((str) => !str.startsWith('#'))
+      .join('\n')
+      .trim(),
```

Comment:
Would it make sense to do all the transformations like they do in nvm:
https://github.com/nvm-sh/nvm/blob/69075e99b0e4b43d31c2fe5360089f308d9cb6b0/nvm.sh#L539